src/components/app/AuthenticatedPage.test.jsx

```diff
@@ -64,14 +67,32 @@ const AuthenticatedPageWrapper = ({
   appContextValue = defaultAppContextValue,
 }) => (
   <IntlProvider locale="en">
-    <AppContext.Provider value={appContextValue}>
-      <AuthenticatedPage>
-        {children}
-      </AuthenticatedPage>
-    </AppContext.Provider>
+    <QueryClientProvider client={queryClient}>
```

Comment:
Is the `queryClient` here intended to be called as a function, i.e., `queryClient()`?